### ISSUE TITLE
calcurse-caldav PasswordCommand trailing '\n'

### ISSUE THREAD

**Issue by eggersn** [lfos/calcurse#473]
**Version information.** 
```
$ calcurse -v
calcurse 4.8.1 -- text-based organizer

Copyright (c) 2004-2023 calcurse Development Team.
This is free software; see the source for copying conditions.
```
Same behavior for commit [95bb55f68b446e4461f27ef3a74a7d2eda950289](https://github.com/lfos/calcurse/commit/95bb55f68b446e4461f27ef3a74a7d2eda950289).

**Bug description.** 
When using the configuration option
```
[Auth]
PasswordCommand = pass calcurse-bug
```
calcurse-caldav.py uses *subprocess.run* to evaluate the provided command. 
This produces a trailing newline character '\n'

**Reproduce.** 
1. generate password
      ``` console
      $ pass generate calcurse-bug 30
      1 file changed, 0 insertions(+), 0 deletions(-)
      create mode 100644 calcurse-bug.gpg
      The generated password for calcurse-bug is:
      QzUyv\8zp8}Gf{<$Xt{IB"1]~#Rc$6
      ```
2. change configuration
      ``` bash
      # Credentials for HTTP Basic Authentication (if required).
      # Set `Password` to your password in plaintext (unsafe),
      # or `PasswordCommand` to a shell command that retrieves it (recommended).
      [Auth]
      Username = Simon
      PasswordCommand = pass calcurse-bug
      ```
3. start tshark and run *calcurse-caldav --init two-way*
      ``` console
      ~ $ tshark -i eth0 -Y http -Tfields -e http.authbasic
      Capturing on 'eth0'
       ** (tshark:718509) 10:57:25.603134 [Main MESSAGE] -- Capture started.
       ** (tshark:718509) 10:57:25.603195 [Main MESSAGE] -- File: "/tmp/wireshark_eth0JS57B2.pcapng"
      Simon:QzUyv\8zp8}Gf{<$Xt{IB"1]~#Rc$6\n
      ```

**Expected Behavior.** calcurse-caldav should strip trailing '\n'.

**Proposed Fix.** use *strip()* to remove trailing '\n'.
```
calcurse (pu)$ git diff
diff --git a/contrib/caldav/calcurse-caldav.py b/contrib/caldav/calcurse-caldav.py
index f9488e6..827ca1e 100755
--- a/contrib/caldav/calcurse-caldav.py
+++ b/contrib/caldav/calcurse-caldav.py
@@ -680,7 +680,7 @@ elif config.get('Auth', 'Password'):
     password = config.get('Auth', 'Password')
 elif config.get('Auth', 'PasswordCommand'):
     tokenized_cmd = shlex.split(config.get('Auth', 'PasswordCommand'))
-    password = subprocess.run(tokenized_cmd, capture_output=True).stdout.decode('UTF-8')
+    password = subprocess.run(tokenized_cmd, capture_output=True).stdout.decode('UTF-8').strip()
 else:
     password = None
```

**Comment by eggersn** [comment]
Meanwhile, a somewhat inelegant mitigation is to use
``` bash
PasswordCommand = bash -c "pass calcurse-bug | tr -d '\n'"
```

**Comment by Maaxxs** [comment]
fixed in d4eb9f85d07351e184ba16ff91c327425653d3e9
